launch_testing_ros/test/examples/wait_for_topic_inject_callback_test.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2025 Open Source Robotics Foundation, Inc.
```

Comment:
If we're renaming "callback" to "trigger," this file should be renamed to `launch_testing_ros/test/examples/wait_for_topic_inject_trigger_test.py`

Reply:
That is true. Thank you.